### PR TITLE
upgrade code to polymer 2.0

### PR DESCRIPTION
**Pull request by varletconst**

### CODE REVIEW

**Comment by varletconst** [comment]
Hi,
Did you get any chance to take a look at this ?
Thanks.

File: demo/index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+    <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+
+    <link rel="import" href="../../iron-demo-helpers/demo-pages-shared-styles.html">
+    <link rel="import" href="../../iron-demo-helpers/demo-snippet.html">
+    <link rel="import" href="../highcharts-chart.html">
+
+    <custom-style>
+        <style is="custom-style" include="demo-pages-shared-styles">
+        </style>
```
Comment (avdaredevil) [inline demo/index.html]: Move closing to the same line

File: highcharts-shared-styles.html [hunk 2]
```diff
@@ -1,10 +1,11 @@
+<link rel="import" href="../shadycss/apply-shim.html">
```
Comment (avdaredevil) [inline highcharts-shared-styles.html]: Is this required?

**Comment by avdaredevil** [comment]
Good to merge! Thanks, man!

File: highcharts-shared-styles.html (hunk 2 above)
Comment (varletconst) [reply highcharts-shared-styles.html]: Yes it is. [here](https://www.polymer-project.org/2.0/docs/upgrade#class-based-elements-import-the-css-mixin-shim)

**Pull request merged**